manifests/pipelines/s3-fetch-pipeline.yaml

```diff
@@ -289,8 +315,14 @@ spec:
     runAfter:
     - test-model-rest-svc
     taskRef:
-      kind: ClusterTask
-      name: openshift-client
+      resolver: git
+      params:
+        - name: url
+          value: https://github.com/tektoncd/catalog.git
+        - name: revision
+          value: main
+        - name: pathInRepo
+          value: task/openshift-client/0.2/openshift-client.yaml
```

Comment:
@MarianMacik I'll update this once this PR is merged in https://github.com/openshift-pipelines/task-openshift/pull/47